### PR TITLE
Streamer: document how to convert to .mp4

### PR DESCRIPTION
**Pull request by darzu**

### CODE REVIEW

File: docs/streamer/docs.md [hunk 1]
```diff
@@ -107,6 +107,27 @@ like 1920x1080. Once your recording is done, click stop to generate the ``.webm`
 The video output and your microphone typically have a delay that needs to be compensated. To compute the delay, record a video and clap!
 Then measure the time between the clap sound and the video. That's your offset!
 
+### File format
+
+Streamer will save your recording as a ".webm" file with the "h264" codec. Many popular video editting tools don't support this, like iMovie. One way to get around this is to convert it into a more standard format like .mp4. To do this, we recommend using a popular command line tool called "ffmpeg".
```
Comment (pelikhan) [inline docs/streamer/docs.md:112]: It's supported by YouTube :)

Comment (pelikhan) [reply docs/streamer/docs.md:112]: and Chrome/Edge itself as well.

Comment (darzu) [reply docs/streamer/docs.md]: Tweaked wording a little

**Comment by pelikhan** [comment]
LGTM

**Pull request merged**